apps/daily-delight/src/app/components/task-calendar.tsx

```
@@ -0,0 +1,75 @@
+                  .map((task) => (
+                    <div
+                      key={task.id}
+                      className="text-xs bg-blue-100 p-1 rounded mb-1 truncate"
+                    >
+                      {task.name}
+                    </div>
+                  ))}
+              </div>
+            </div>
+          ) : (
+            <div className="aspect-square p-2 rounded-md"></div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)